contracts/ArrowVestingBase.sol

```diff
@@ -1,21 +1,22 @@
-pragma solidity ^0.8.0;
-
 // SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
 
-import "@openzeppelin-upgradeable/contracts/finance/VestingWalletUpgradeable.sol";
+// Valid import paths for hardhat (need to comment out when using brownie)
+import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol";
 
-/**
-    Base implementation of vesting contract that will be cloned into individual vesting wallets pointing at specific beneficiary addresses.
- */
+/// @title Base implementation of vesting contract that will be cloned into individual vesting wallets pointing at specific beneficiary addresses.
 contract ArrowVestingBase is VestingWalletUpgradeable {
-
     constructor() initializer {}
 
     function initialize(
-        address beneficiaryAddress,
-        uint64 startTimestamp,
-        uint64 durationSeconds
+        address _beneficiaryAddress,
+        uint64 _startTimestamp,
+        uint64 _durationSeconds
     ) public initializer {
-        __VestingWallet_init(beneficiaryAddress, startTimestamp, durationSeconds);
-    } 
+        __VestingWallet_init(
+            _beneficiaryAddress,
+            _startTimestamp,
+            _durationSeconds
```

Comment:
Let's keep everything in this PR relevant to upgradable contracts. We can open up another PR to fix variable name styling if necessary later to keep everything compartmentalised.